tests/integration/webserver.py

```diff
@@ -104,6 +111,17 @@ def __eq__(self, other):
         else:
             return NotImplemented
 
+    def __lt__(self, other):
+        if isinstance(other, (Request, ExpectedRequest)):
+            return (self.verb, self.path) < (other.verb, other.path)
+        else:
+            return NotImplemented
+
+    def __str__(self):
+        return '<ExpectedRequest {} "{}">'.format(self.verb, self.path)
```

Comment:
I'd prefer something like `'ExpectedRequest(verb={!r}, path={!r})'.format(self.verb, self.path)` to follow the "`eval(repr(x)) == x` guideline if possible.